TestResolver is flaky

```
--- FAIL: TestResolver-4 (0.00s)
	resolver_test.go:89: didn't Stop in time
FAIL
```

Seen 2-3 times recently, including: https://circleci.com/gh/weaveworks/scope/1100

--- a/probe/resolver_test.go
+++ b/probe/resolver_test.go
@@ -85,7 +85,7 @@ func TestResolver(t *testing.T) {
 	go func() { r.Stop(); close(done) }()
 	select {
 	case <-done:
-	case <-time.After(time.Millisecond):
+	case <-time.After(100*time.Millisecond):
 		t.Errorf("didn't Stop in time")
 	}
 }

